requirements.txt

```diff
@@ -8,3 +8,4 @@ huggingface_hub==0.32.4
 onnx==1.18.0
 onnxscript==0.3.2
 tqdm==4.66.6
+safetensors~=0.4.3
```

Comment:
Cool, I didn't know this wasn't provided with the HuggingFace Hub package. Should this dependency also be added to `pyproject.toml`?

Reply:
I am not aware of pyproject.toml but you might be right. Remember, I don't know Python...yet.